x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_lifecycle/hooks/use_ingestion_rate.tsx

```diff
@@ -60,16 +65,23 @@ export const useIngestionRate = ({
 
   const ingestionRateFetch = useStreamsAppFetch(
     async ({ signal }) => {
-      if (!stats) {
-        return;
-      }
-
       const intervalData = getIntervalAndType(timeState, core);
       if (!intervalData) {
         return;
       }
+
+      const start = moment(timeState.start);
+      const end = moment(timeState.end);
+
       const { interval, intervalType } = intervalData;
 
+      if (!stats) {
+        return { start, end, interval, buckets: {} };
+      }
+      const indexName = isFailureStore
+        ? `${getFailureStoreIndexName(definition)}${FAILURE_STORE_SELECTOR}`
```

Comment:
for wired stream `logs.child` this would output
`logs.child, logs.child.*::failures` but it should instead be
`logs.child::failures, logs.child.*::failures` ?

Reply:
Yes, correct